vendor/brave-ios/Ads/BATBraveAds.h

```diff
@@ -113,7 +113,7 @@ NS_SWIFT_NAME(BraveAds)
 
 /// Report that a page has loaded in the current browser tab, and the inner text
 /// within the page loaded for classification
-- (void)reportLoadedPageWithURL:(NSURL *)url innerText:(NSString *)text;
+- (void)reportLoadedPageWithURL:(NSURL *)url innerText:(NSString *)text tabId:(NSInteget)tabId;
```

Comment:
Typo on NSIntege**r**, will fail to compile

Also on the implementation in the mm file

Reply:
Fixed